### ISSUE TITLE
CoupledSDEs documentation

### ISSUE THREAD

**Issue by JonasKoziorek** [JuliaDynamics/DynamicalSystemsBase.jl#219]
**Describe the bug**
I think that the `CoupledSDEs` part of the documentation should be better incorporated into the rest of the docs. Other types of dynamical systems such as `CoupledODEs` or `PoincareMap` have their documentation under DynamicalSystemsBase.jl section but `CoupledSDEs` have their own section which break this pattern.

**Example**
![image](https://github.com/user-attachments/assets/3f0cb8ff-2136-49dc-991a-58e8455bf6f5)

**Comment by Datseris** [comment]
Yeah, probably we need to rethink this a bit. At the moment the standard tutorial/documentation for how to make a typical system `CoupledODEs/Map` is in the main DynamicalSYstems.jl tutorial. Here only the docstrings are expanded. Because the SDE version is significantly more complex, we felt it needed a dedicated doc page. And it does. 

So, ideally, one would have full proper documentation for all types of dynamical systems here. But I simply haven't had the time to contribute it (and honestly it is probably low priority; the docstrings are very good).
